app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+# from app.errors import * bad practice or something?
+from app.errors import NotVaccinatedError, OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    without_mask = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            without_mask += 1
+    if without_mask > 0:
+        return f"Friends should buy {without_mask} masks"
+    else:
```

Comment:
And do not use `else` condition because if the previous condition is not True you will return this expression anyway, just write `return f"Friends can go to {cafe.name}"` and shift it to the left by 4 spaces

Reply:
You're absolutely right. Thank you!